mettle/src/stdapi/sys/memory.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _STDAPI_MEMORY_H
+#define _STDAPI_MEMORY_H
+
+#ifdef __linux__
```

Comment:
Any access to objects/signatures defined in this ifdef will need to have a matching ifdef in `memory.c`.